### ISSUE TITLE
Definition of equilibrium solution

### ISSUE THREAD

**Issue by ngiann** [jmzobitz/ModelingWithR#13]
Please excuse me for posting this comment that may appear nit-picking:

In page 67, section 5.2, the definition of an equilibrium solution is given.

What I found a bit confusing (but finally worked it out for myself) is the limit: 

lim_{t->inf) y(t) = y*

This limit returns a scalar value, but we are equating to a constant function.
I found this confusing, but after one day, I managed to understand what the text means.
Now I do understand that the limiting value should be equal to the constant value of the constant function.

Obviously, the book emphasises intuition (which is why I am reading it), but I thought I should drop a line about this in any case.

Please free to ignore this issue and close it without replying, if you think it's not helpful.

**Comment by jmzobitz** [comment]
@ngiann Thanks!  This is helpful feedback.  I noted this on the running list of [errata](https://docs.google.com/document/d/1BJmyeQbrDRmUdoM11pZTA_kTV0XG76L1UQLOxCIR8M0/edit?usp=sharing).

**Comment by jmzobitz** [comment]
Getting to these revisions, here is my rephrasing - I am updating this and merging it into an update and conversion of the text to quarto.  Thanks for the nudge!

The **stability** of an equilibrium solution describes the long-term behavior of the family of solutions to a differential equation. Some solutions may converge to the equilibrium in the long run, while others may not. More formally stated:

> The constant function $y=y_{*}$ is considered an asymptotically *stable* solution to a differential equation $\displaystyle \frac{dy}{dt} = f(y)$ if there exists a solution $y_{S}(t)$, such that $\displaystyle \lim_{t \rightarrow \infty} y_{S}(t) = y_{*}$.

In other words, as time increases, the the family of solutions to the differential equation is approximated by the constant function $y=y_{*}$. You may note that the definition of stability relies on determining the solution $y_{S}(t)$. However we can circumvent determining this solution by using ideas from calculus and the rate of change:
